source/acquaman/AMGenericScanActionControllerAssembler.cpp

```diff
@@ -156,7 +168,13 @@ AMAction3* AMGenericScanActionControllerAssembler::generateActionTreeForStepAxis
 
 	if (axisControl){
 
-		AMAction3 *regionStart = AMActionSupport::buildControlMoveAction(axisControl, stepScanAxisRegion->regionStart());
+		AMAction3 *regionStart = 0;
+
+		if (direction_ == AMScanConfiguration::Increase)
```

Comment:
Yes, it is possible.  But it's always been possible (for instance you could put in 400 eV to 300 eV and just have the worst scan ever for the longest time.